### PR TITLE
msx1_flop.xml: Added 11 items, 8 working.

### PR DESCRIPTION
**Pull request by wilbertpol**
Replace TT Racer (Netherlands) with a better dump. [file-hunter]
Replace Zanac (Netherlands) with a better dump. [file-hunter]

New working software list items
-------------------------------
Aackotext II (Netherlands, v2.2) [file-hunter]
Brasil Geográfico (Brazil) [file-hunter]
dBASE II (Netherlands) [file-hunter]
MS Text (Netherlands) [file-hunter]
Score Editor (Japan) [file-hunter]
MSX-AIDS (Japan, v1.1) [file-hunter]
Nihongo Waupuro Kan-juku Tomato (Japan) [file-hunter]
JUBILEUM Diskette (Netherlands) [file-hunter]

New NOT_WORKING software list additions
------------------------------------------
Serial Interface (Netherlands) [file-hunter]
MSX Data Communications (Netherlands) [file-hunter]
Yamaha Portatone PSR-6300 Demonstration [file-hunter]

### CODE REVIEW

File: hash/msx1_flop.xml [hunk 1]
```diff
@@ -1253,6 +1319,37 @@ The following floppies came with the machines.
...
+		<publisher>Toshiba-EMI Ltd.</publisher>
+		<info name="alt_title" value="ＭＳＸ－Ａｕｄｉｏシリーズスコアエディタ"/>
+		<info name="serial" value="PS-3009S"/>
+		<info name="barcode" value="4988006014817"/>
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="368640">
+				<rom name="msx audio series score editor - toshiba-emi.dsk" size="368640" crc="db18b408" sha1="25f1650f78e0842599d2adf730864dbec5281107"/>
+			</dataarea>
+		</part>
+	</software>
```
Comment (cuavas) [inline hash/msx1_flop.xml:1351]: Is this the cover of this one? https://msx.jpn.org/tagoo/s_check.cgi?LINE=907&PHOTO=1

Where does the title used in the description and `alt_title` come from?

The box seems to show “MSX-Audio対応簡単作曲ソフト” in the blue bubble (roughly “MSX-Audio-compatible easy composition software”), but the title seems to just be “Score Editor”/“スコア・エディタ”.

File: hash/msx1_flop.xml [hunk 2]
```diff
@@ -1366,6 +1474,28 @@ The following floppies came with the machines.
 		</part>
 	</software>
 
+	<software name="kanjukut">
+		<description>Nihongo Waupuro Kan Juku Tomato (Japan)</description>
+		<year>1985</year>
+		<publisher>Sony</publisher>
+		<info name="alt_title" value="日本語ワープロ漢熟トマト"/>
```
Comment (cuavas) [inline hash/msx1_flop.xml:1481]: I’d make “Kanjuku” one word, or hyphenate it as “Kan-juku”.  漢熟 is an abbreviation for 漢字習熟度, as in 漢字習熟度検定 (kanji shūjuku-do kentei, kanji proficiency test).

File: hash/msx1_flop.xml (hunk 1 above)
Comment (wilbertpol) [reply hash/msx1_flop.xml]: Yes, that is the one. Just "Score Editor" does make more sense.

File: hash/msx1_flop.xml (hunk 2 above)
Comment (wilbertpol) [reply hash/msx1_flop.xml]: Thanks, changed to Kan-juku.

**Pull request merged**